.github/workflows/release-on-milestone-closed-triggering-release-event.yml

```diff
@@ -0,0 +1,61 @@
+# Alternate workflow example.
```

Comment:
Only change I made is to remove the changelog bump, since we don't maintain a changelog here...